features/accommodations/creation/AmenitiesForm.tsx

```diff
@@ -3,7 +3,7 @@ import { type Amenity } from "../AccommodationModels";
 import { type AmenitiesList, type CreateAccommodation } from "../createAccommodation";
 
 type AmenitiesFormProps = {
-  allAmenities: Amenity[];
+  allAmenities: Map<string, Amenity[]>;
```

Comment:
Is there any particular reason you choose to use map instead of object.

Reply:
There is no particular reason, I usually use a Map when grouping objects

Reply:
Okay. Maps have better performances than objects but syntax around them is a bit uglier 😄